rtrlib/transport/ssh/ssh_transport.c

```diff
@@ -73,7 +73,12 @@ int tr_ssh_open(void *socket)
         goto error;
     }
 
-    const int rtval = ssh_userauth_publickey_auto(ssh_socket->session, NULL, NULL);
+    const int rtval;
+#ifdef LIBSSH_060
+    rtval = ssh_userauth_publickey_auto(ssh_socket->session, NULL, NULL);
+#elif defined LIBSSH_052
```

Comment:
if its not libssh 0.6.x (or above) it has to be 0.5.x, otherwise `cmake` would have failed first. So you can simplify this to `#else` - and may add a short comment something like:
```
#ifdef LIBSSH_060
...
#else // LIBSSH_050
...
#endif
```